### DISCUSSION TITLE
What exactly does it mean by finishing the user profile with full name

### DISCUSSION THREAD

**Discussion by ChrisKeyber** [community/community#55180] [GitHub Education]
"You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio."

I got all my ids, email and I even changed the public name, what does it actually want? And do I need to change exactly? Change my username that used for tagging?

**Comment by ChrisKeyber** [comment]
Screw this garbage I ain't that poor to pay 10 bucks

**Reply by stretch07** [reply]
lmao

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
